react/index.native.js

```diff
@@ -1,3 +1,5 @@
+import 'es6-symbol/implement'; // Polyfill Symbol
```

Comment:
You'll have to polyfill Symbol on web as well because of IE11. I checked on caniuse and locally and Symbol is not supported yet on IE11.